docusaurus.config.js

```diff
@@ -194,10 +194,6 @@ module.exports = {
             to: "/user-entity",
             from: ["/how-to/add-delete-user-entity"]
           },
-          {
```

Comment:
Hey @mulygottlieb,

We moved the Docker Desktop article from /deploy to /deploy/docker-desktop/ when we added more deployment guides. I now want to use /deploy for the main Deployment Overview guide. I had to remove the redirect because Docusaurus was causing build issues.